### PR TITLE
2024 10 28 matriz de riesgo secciones

### PR DESCRIPTION
**Pull request by edburgoss**
## Describe the PR content and subject

En la Matriz de Riesgo la secccicon de KPI de cliente fue dividido en distintas secciones para una mejor visual y busqueda de información

## Type of change

- [x] New feature (non-breaking change which adds functionality)

image

## Checklisk
 
- [x]  Ensure the code follows the project's coding standards
    - [x]  Choose descriptive and unambiguous names.
    - [x]  Make meaningful distinctions.
    - [x]  Use pronounceable names.
    - [x]  Use searchable names.
    - [x]  Replace magic numbers with named constants. (Also magic text)
    - [x]  Avoid encodings. Don’t append prefixes or type information.
- [x]  Check if the new code has proper unit tests
- [x]  Verify that the code is properly commented for understandability
    - [x]  Remove poor quality, not meaningful comments
    - [x]  Use comments on non standard code and hard
- [x]  Check if the code changes are relevant to the pull request description
- [x]  Ensure the new code does not introduce any new bugs
- [x]  Check if the pull request contains minimal and logical commits
- [x]  Check if the code changes affect performance and suggest improvements if it does
- [x]  Review if the pull request resolves the issue it is linked to
    - [ ]  If not, describe why the PR y needed/split
- [x]  Do not include extensive refactoring if not requested
- [x]  Ensure all continuous integration checks are passed
- [x]  Provide proof of completion on issue or pull request

## Summary by CodeRabbit

- **New Features**
	- Introduced two new reports in the "Matriz de Riesgo": "Boletín Concursal" and "Poder Judicial," enhancing client risk management.
	- Added a new blog post detailing improvements in customer KPIs, now organized into three sections for better clarity and analysis.

- **Documentation**
	- New markdown files created to provide detailed information on the updates and enhancements in the risk matrix.

### CODE REVIEW

File: _posts/2024-10-21-reporte-boletin-poder-matriz.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Potenciando la Matriz de Riesgo con Boletines Concursales y Poder Judicial.
+subtitle: Matriz de Riesgo - Cliente.
+gh-repo: /capitalexpress/capitalexpress.github.io
```
[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-21-reporte-boletin-poder-matriz.md:5]: resolver

[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-21-reporte-boletin-poder-matriz.md:5]: validar si es necesario aplicar.

File: _posts/2024-10-21-reporte-boletin-poder-matriz.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ ### Boletín Concursal
+
+Este reporte lo podrán encontrar en la parte inferior de la matriz de riesgo. Una vez desplegado encontrarán la información actualizada del cliente seleccionado.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/boletinconcursal.png" alt="boletinconcursal">
+</p>
+ 
+### Poder Judicial
+Al igual que el Boletín Concursal, estará en la parte inferior de la Matriz de riesgo, una vez desplegada la información encontraras los `Casos Civiles`, `Casos Laborales` y `Casos de Cobranza`. Estos apartados están divididos en distintas tablas para su fácil distinción y búsqueda.
```
[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-21-reporte-boletin-poder-matriz.md:24]: Corregir faltas de ortografía.

File: _posts/2024-10-28-matriz-riesgo-seccion-cliente.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Simplificando el Análisis: Nuevas Secciones en el KPI de cliente en la matriz de riesgo.
+subtitle: Matriz de Riesgo - Cliente.
+gh-repo: /capitalexpress/capitalexpress.github.io
```
[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-28-matriz-riesgo-seccion-cliente.md:5]: validar si es necesario aplicar.

File: _posts/2024-10-28-matriz-riesgo-seccion-cliente.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+gh-badge: [star, fork, follow]
+tags: [ riesgo, creditos , clientes, ejecutivos, operaciones ]
+comments: false
+author: Erick Burgos
+---
+
+## Nuevas Secciones en el KPI de cliente
+
+
```
[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-28-matriz-riesgo-seccion-cliente.md:14]: Aplicar cambios sugeridos, revisa si tienes todas las extensiones que utilizamos como EsLint.

File: _posts/2024-10-28-matriz-riesgo-seccion-cliente.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Nos complace anunciar una mejora significativa en la Matriz de Riesgo de Advance: la separación de los KPI del cliente en tres apartados distintos: `Casos Civiles`, `Documentos` y `Cesiones`. Esta reestructuración permitirá a los usuarios acceder de manera más rápida y eficiente a la información relevante, facilitando un análisis más claro y una mejor toma de decisiones. Acompáñanos a explorar cómo estas nuevas categorías optimizan tu experiencia en Advance.
+
+
+<video width="100%"  controls autoplay loop>
+  <source src="https://cdn.capitalexpress.cl/video/2024-10-28-Matriz-cliente-KPI.mp4" type="video/mp4">
+</video>
```
[the remark this answers is not in the document]
Comment (mlisboa-capex) [reply _posts/2024-10-28-matriz-riesgo-seccion-cliente.md:20]: validar si te solicitaron que la reproducción sea en loop, si no sería bueno aplicar este cambio.

[the remark this answers is not in the document]
Comment (edburgoss) [reply _posts/2024-10-28-matriz-riesgo-seccion-cliente.md:20]: El cambio fue solicitado, no merece aplicar el cambio

**Review by arubilar-capex — APPROVED** [review]

**Review by mlisboa-capex — APPROVED** [review]
aprobado!

**Review by arubilar-capex — APPROVED** [review]

**Pull request merged**